auacm/app/static/js/controllers/TeamController.js

```diff
@@ -80,8 +80,7 @@ app.controller('TeamController', ['$scope', '$http', '$location', '$window',
         });
     };
 
-    var socket = io.connect('http://' + $window.location.host + '/register');
-    socket.on('connect', function() {});
+    var socket = new Socket('ws://' + $window.location.host + '/websocket');
```

Comment:
Since we are, at the moment, a single page application, is there really a need to open connections multiple times? Before we had separate domains, but that may not be necessary and that doesn't seem to be what were doing anymore. Perhaps one `appSocket` defined in MainController that can be accessed by all the subcontrollers. I don't know if that would cause extra overhead or something.